pkgs/development/python-modules/brotlicffi/default.nix

```
@@ -24,7 +24,10 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     cffi
-  ];
+  ] ++ (if pythonOlder "3.4" then
+    [ enum34 ]
+  else
+    []);
```

most common way of expressing this.
```suggestion
  ] ++ lib.optionals (pythonOlder "3.4") [
    enum34
  ];
```

---

Thanks! Implemented